components/system-broker/pkg/http/types.go

```diff
@@ -21,6 +21,9 @@ import (
 	"time"
 )
 
+const TokenExpirationBuffer = 10 * time.Second
```

Comment:
maybe we can have this configurable ? not sure if it makes sense though

Reply:
Probably it makes sense to be reused request timeout for this. Since this is the time that left for the token to be valid and if it is smaller than the request timeout it will expire during a request and if it is longer it does not make sense to refresh the token since it has the validity for the max request duration. What do you think?